commands/scp.go

```diff
@@ -177,7 +177,7 @@ func getInfoForScpArg(hostAndPath string, hostInfoLoader HostInfoLoader) (h Host
 	}
 
 	if h.GetSSHKeyPath() != "" {
-		args = append(args, "-o", fmt.Sprintf("IdentityFile=%s", h.GetSSHKeyPath()))
+		args = append(args, "-o", fmt.Sprintf(`IdentityFile="%s"`, h.GetSSHKeyPath()))
```

Comment:
I just realized that a possibly better fix for this would be to use `%q` instead of `%s`, which would also take care of the odd case where the path itself contains quotes. WDYT?

Reply:
@shin- Agreed. Changes made.